[2020-06-17 19:04:14,811] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: generator_v0.generate_file 2020-06-17T03:30:00+00:00 [None]>
[2020-06-17 19:04:14,823] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: generator_v0.generate_file 2020-06-17T03:30:00+00:00 [None]>
[2020-06-17 19:04:14,823] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-17 19:04:14,823] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-17 19:04:14,823] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-17 19:04:14,990] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): generate_file> on 2020-06-17T03:30:00+00:00
[2020-06-17 19:04:14,992] {standard_task_runner.py:53} INFO - Started process 5524 to run task
[2020-06-17 19:04:15,183] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: generator_v0.generate_file 2020-06-17T03:30:00+00:00 [running]> strx
[2020-06-17 19:04:15,194] {logging_mixin.py:112} INFO - Generating  1592435055 ...
[2020-06-17 19:04:15,194] {logging_mixin.py:112} INFO - Finished.
[2020-06-17 19:04:17,196] {python_operator.py:114} INFO - Done. Returned value was: _
[2020-06-17 19:04:17,497] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=generator_v0, task_id=generate_file, execution_date=20200617T033000, start_date=20200617T230414, end_date=20200617T230417
[2020-06-17 19:04:24,669] {logging_mixin.py:112} INFO - [2020-06-17 19:04:24,668] {local_task_job.py:103} INFO - Task exited with return code 0
